python3/vim_pandoc/helpparser.py

```diff
@@ -116,7 +116,7 @@ def wrap_flag(flag):
             extra = {}
             extra['action'] = 'store_true' if not opt.arg else 'store'
             # some options can be given several times
-            if any(map(lambda x: x.isupper() and x != 'T' or x == 'bibliography', opt.names)):
+            if any(map(lambda x: x.isupper() and x != 'T' or x == 'bibliography' or x == 'defaults', opt.names)):
```

Comment:
what about

``` python
repeatable_options = ['bibliography', 'defaults']
if any(map(lambda x: x.isupper() and x != 'T' or x in repeatable_options, opt.names)):
```

?

Reply:
Ok, I went thru the manual and added to your list in the next commit.

Reply:
Excellent! Thanks so much.